yarn-project/simulator/src/client/client_execution_context.ts

```diff
@@ -382,9 +361,8 @@ export class ClientExecutionContext extends ViewDataOracle {
    * @param counter - The effects counter.
    */
   public override emitEncryptedNoteLog(noteHash: Fr, encryptedNote: Buffer, counter: number) {
-    const encryptedLog = new CountedLog(new EncryptedL2NoteLog(encryptedNote), counter);
+    const encryptedLog = this.noteCache.addNewLog(encryptedNote, counter, noteHash);
```

Comment:
Do we still need to add it to noteCache? Think nothing is getting logs from it now.

Reply:
Correct - I only left it in so we could still use `getAllNoteEncryptedLogs` in future. It's not currently being used now, and we'd just use the `collect` method on the result for checking this if we wanted I think? So I'll remove the `logs` map here unless there are any objections!

Reply:
Sounds good! Will be nice to have less code to maintain if it's not going to be used any time soon :)